jobs/BaseJob.php

```diff
@@ -47,12 +47,53 @@ abstract class BaseJob implements ShouldQueue
      */
     public $queue;
 
+    /**
+     * The number of times the job may be attempted.
+     *
+     * @var int
+     */
+    public $tries = 3;
+
+    /**
+     * The number of SECONDS to wait before retrying the job.
+     *
+     * @var int
```

Comment:
Rather than `@var` self-documentation, better to just use the `public int $backoff = 5;` directly. (Also below)

Reply:
Done .